### PR TITLE
add navbar

### PR DESCRIPTION
**Pull request by amiqbal7**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,37 @@
...
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#testi">Testimonials</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#faq">FAQ</a>
+            </li>
+          </ul>
+          <div class="d-flex">
+            <button class="btn btn-outline-success bg-success text-white" type="submit">Register</button>
```
Comment (CoreX20) [inline index.html:32]: change button color to blue

Comment (CoreX20) [inline index.html]: change button color to blue

**Review by CoreX20 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (amiqbal7) [reply index.html]: okey

**Review by amiqbal7 — COMMENTED** [review]
Done, please merge pull req

**Pull request merged**